nibabel/tests/test_parrec.py

```diff
@@ -366,6 +389,15 @@ def test_sorting_multiecho_ASL():
     # check that slices vary fastest
     assert_array_equal(sorted_slices[:nslices], np.arange(1, nslices+1))
 
+    # check volume labels
+    vol_labels = asl_hdr.get_volume_labels()
+    assert_equal(sorted(list(vol_labels.keys())),
```

Comment:
Can drop `sorted` and test for order `['echo number', 'label type', 'dynamic scan number']`